### PR TITLE
Add digital ocean guide

### PR DESCRIPTION
**Pull request by lucywyman**

### CODE REVIEW

File: content/guides/digital_ocean.mkd [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   sensible ("Schoolwork", "Learning Linux", "IRC", etc.), then choose
+   the $5/month sized machine -- this will more than likely be 
+   more than enough space for right now.
+7. Select either the San Francisco or New York Regions, and the 
+   default ubuntu version.  
+8. If you would like for your droplet to be even more secure, you can 
+   create an ssh key by running ``ssh-keygen -t rsa -b 4096 -C "your_email@example.com"``
+   and add the public key (copy-paste it) to DO.  This is encouraged!
+9. Click "Create Droplet", and you're good to go!  You can now run
+   ``ssh root@<IP Address>`` and either enter your password or
```
Comment (dspt) [inline content/guides/digital_ocean.mkd]: You might mention that they will email the password.

**Comment by dspt** [comment]
Including a note about setting up a $website.me domain via the Student Pack could help them so they don't have to remember the IP address.

**Comment by lucywyman** [comment]
@dspt Should setting up your own domain be it's own guide?  I don't know that I can whip that up super quick, but in starting to write it, I think it's a little lengthy to include here :/

**Comment by lucywyman** [comment]
@ElijahCaine ?

File: content/guides/digital_ocean.mkd [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Your digital ocean droplet will have an IP address that you use to ssh
+to.  IP addresses tend not to be very human-friendly, so there are a 
+few things you can do so you don't have to remember it:
+
+1. Use an alias. If you're in bash, open up your ``bashrc`` and add
+
+    alias linux="ssh root@<IP Address>"
+
+   to the bottom of it.  Then you only need to run ``$ linux`` to
```
Comment (ElijahCaine) [inline content/guides/digital_ocean.mkd]: Change ``$ linux`` to "the command ``linux`` in your terminal". There's been plenty of confusion about the ``$ command`` (typing the ``$`` into the terminal and such), we can avoid the confusion all together by just avoiding that syntax sometimes.

**Comment by ElijahCaine** [comment]
@lucywyman 
1. I think a separate 'setting up your own website with the GH Student Pack' would be good, they could link to each other.
2. I would like to add pictures to this guide. If you give me some time later today, I'll push some images to this branch that contextualize the guide a bit more. If you don't think it's necessary that's fine, I just know images can be *very* helpful for newbies.

**Pull request merged**